ui/admin/app/templates/application.hbs

```diff
@@ -70,42 +70,76 @@
                 </dd.Interactive>
               </Hds::Dropdown>
             {{/if}}
-
-            <Hds::Dropdown
-              @enableCollisionDetection={{true}}
-              data-test-side-nav-user-dropdown
-              as |dd|
-            >
-              {{#if this.session.data.authenticated.username}}
+            {{#if this.session.data.authenticated.username}}
```

Comment:
In what scenarios would this not be true?

Reply:
after login when the application route renders the side nav isAuthenticated is true but the value of username has not been set yet.